src/toast.ts

```diff
@@ -0,0 +1,144 @@
+import {WorkspaceSvg} from 'blockly';
```

Comment:
I have put no effort into doing things in a Blockly-like way here at this point, or even finding out what that might be.